src/UI/CredentialsImportTooltip.js

```diff
@@ -0,0 +1,39 @@
+import HTMLTooltip from './HTMLTooltip.js'
+
+class CredentialsImportTooltip extends HTMLTooltip {
+    /**
+     * @param {import("../DeviceInterface/InterfacePrototype.js").default} device
+     */
+    render (device) {
+        this.device = device
+
+        this.shadow.innerHTML = `
+${this.options.css}
+<div class="wrapper wrapper--data top-autofill wrapper--credentials-import" hidden>
+    <div class="tooltip tooltip--data">
+        <button class="tooltip__button tooltip__credentials-import-button js-promo-wrapper">
+            <div class="tooltip__credentials-import-content">
+                <div class="tooltip__credentials-import-icon"></div>
+                <div class="tooltip__credentials-import-text-container">
+                    <h1 class="tooltip__credentials-import-heading">Import password to DuckDuckGo...</h1>
```

Comment:
These strings should be localized using [`this.device.t`](https://github.com/duckduckgo/duckduckgo-autofill/blob/b629ed523510bd0e02611733f45a4e33f6ab4410/src/DeviceInterface/InterfacePrototype.js#L96) 😉.